core/src/test/java/hudson/logging/LogRecorderTest.java

```diff
@@ -115,6 +115,7 @@ private static LogRecord createLogRecord(String logger, Level level, String mess
         return r;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Would rather you _not_ suppress the warning here. Either fix the usage (should now be `matches`) or leave it alone.

Reply:
I think this test is specifically testing the deprecated function.